### ISSUE TITLE
docker-compose command not found

### ISSUE THREAD

**Issue by The-Robin-Hood** [documenso/documenso#175]
Issue Statement:

The package json script contains a docker-compose command that is v1 docker compose command which in v2 it is switched to "docker compose". This inconsistency can cause errors and delays in the installation process, especially for novice users who are unfamiliar with Docker.

Recommendation:

To resolve this issue, it is suggested that the package.json script should be updated to include both versions of the command to accommodate all systems.

**Comment by Mythie** [comment]
Yep this is going to be because I'm a tech boomer 😂 

Appreciate the issue and the PR to resolve

**Comment by nikmartin** [comment]
the patch doesn't work on linux. I'll see if I can find a workaround

**Comment by The-Robin-Hood** [comment]
@nikmartin whats the prob you are facing ? Coz it did worked for me

**Comment by nikmartin** [comment]
running this in bash: 
` docker compose -f ./docker/compose-without-app.yml || docker-compose -f ./docker/compose-without-app.yml up -d`

returns:

```Usage:  docker compose [OPTIONS] COMMAND

Docker Compose
...
```

BUT adding the full command to each option: 
 `docker compose -f ./docker/compose-without-app.yml up -d || docker-compose -f ./docker/compose-without-app.yml up -d`

works for me

**Comment by The-Robin-Hood** [comment]
One way to fix is 

```
"docker:compose-up": "docker compose -f ./docker/compose-without-app.yml up -d || docker-compose -f ./docker/compose-without-app.yml up -d",
"docker:compose-down": "docker compose -f ./docker/compose-without-app.yml down || docker-compose -f ./docker/compose-without-app.yml down"
 ```
 
 and remove docker-compose

**Comment by Mythie** [comment]
Sounds like the way to go! Sorry for the delay, you can either raise a PR or I can do so shortly :)

**Comment by Mythie** [comment]
Resolved with #184
